src/components/Link/Link.scss

```diff
@@ -19,7 +19,6 @@
 
   &:focus {
     $chrome-default-outline: rgba(0, 103, 244, 0.247) auto rem(4.5px);
-    outline: var(--p-override-none);
```

Comment:
We may still need `outline: none` here?

Reply:
I made sure to check all of these style changes on Chrome, Safari, and Firefox and didn't see an outline on focus for this one but would love it if you could confirm that's the case for you too!

Reply:
This is being overridden with our styles so we should be good 👍